the-open-university-harvard.csl

```diff
@@ -44,6 +44,11 @@
       </substitute>
     </names>
   </macro>
+  <macro name="translator">
+    <names variable="translator">
```

Comment:
instead of hard-coding the label below, use `<label form="short" suffix=" "/>` here -- potentially with `plural="always"` to always get trans.